src/directives/formio.js

```diff
@@ -155,6 +155,17 @@ module.exports = function() {
           }
         });
 
+        $scope.$on('iframe-getIframePositions', () => {
```

Comment:
This doesn't compile because we do not run this through babel on 2.x branch. I am not sure how you tested this.